homeassistant/components/axis/strings.json

```diff
@@ -13,13 +13,13 @@
       }
     },
     "error": {
-      "already_configured": "Device is already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "Config flow for device is already in progress.",
```

Comment:
Already in progress is also a common string

Reply:
I couldn't find it in homeassistant/strings.json... where is it defined?

Reply:
"already_in_progress": "Configuration flow is already in progress",
https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json#L60

Reply:
It is not exactly the purpose of the changes, but these keys should be in the "abort" section

Reply:
> "already_in_progress": "Configuration flow is already in progress",
> 
> https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json#L60

Ok yes I see. That was only added in the last 24h so missing from my WC!

Reply:
Living in the fast lane :)